src/smart-components/Channel/components/SuggestedMentionList/SuggestedUserMentionItem.tsx

```diff
@@ -4,37 +4,47 @@ import Avatar from '../../../../ui/Avatar';
 import Label, { LabelTypography, LabelColors } from '../../../../ui/Label';
 import { LocalizationContext } from '../../../../lib/LocalizationContext';
 import SendBird from 'sendbird';
+import { MentionItemUIEvent } from '../../../..';
 
 interface SuggestedUserMentionItemProps {
-  member: SendBird.User;
+  member: SendBird.User | SendBird.Member;
```

Comment:
We need same change on https://github.com/sendbird/sendbird-uikit-react/blob/develop-v3/scripts/index_d_ts#L669:L669